backend/services/PostHandler.py

```
@@ -0,0 +1,65 @@
+        query = query.filter(Post.user_id == user_id)
+    post = query.first()
+    if not post:
+        raise HTTPException(status_code=404, detail=STATUS_404_ERROR)
+    return post
+
+
+def update_post_content(post: Post, content: Optional[str]):
+    if content is not None:
+        post.content = content
+
+import re
```

Ruff (0.8.2)

Module level import not at top of file

(E402)